### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by C-O-P**
1.upgrading the tab "Mozilla",fixing the position so that clicked tabs remains even when page is scrolled down.
2.styling the font "Mozilla" .Expanding the word when cursor is brought over it.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -18,8 +18,15 @@ html {
 }
 
 #tabzilla {
+	position:fixed;
```
Comment (C-O-P) [inline style.css]: fixing the tab even when page is scrolled down

File: style.css [hunk 2]
```diff
@@ -18,8 +18,15 @@ html {
...
     margin-right: 100px;
 }
+/* Actual upgradation
+*  Adding new style on hovering
+*/
+#tabzilla:hover{
```
Comment (C-O-P) [inline style.css]: expands the font when cursor is brought over

Comment (jdm) [inline style.css]: How does it expand the font? It's just changing the color property.

Comment (C-O-P) [inline style.css]: apologies for my wrong words.And I would like to work further on it.

**Comment by jdm** [comment]
Sorry, I don't think these changes improve the experience of using the website. Thanks for contributing them, however!

**Comment by C-O-P** [comment]
No problem.

**Pull request closed without merging**